setup.py

```diff
@@ -29,19 +29,20 @@ def run(self):
         raise SystemExit(errno)
 
 
-setup(name='sklearn-pandas',
+setup(name='sklearn-pandas2',
       version=__version__,
       description='Pandas integration with sklearn',
-      maintainer='Israel Saeta Pérez',
-      maintainer_email='israel.saeta@dukebody.com',
-      url='https://github.com/paulgb/sklearn-pandas',
+      maintainer='Ritesh Agrawal',
```

Comment:
Agree to change the maintainer, but perhaps you want to keep the package name as `sklearn-pandas` ;)

Reply:
I changed name assuming no-one will respond from sklearn-pandas group and I will then just keep updating on my fork. But I will change it back to sklearn-pandas now.